src/main/java/io/ark/core/crypto/Crypto.java

```diff
@@ -23,6 +28,18 @@
 
   private static ECNamedCurveParameterSpec curve = ECNamedCurveTable.getParameterSpec("secp256k1");
 
+  public static List<String> getPassphrase() {
+    byte[] entropy = new byte[16];
+    List<String> passphrase = new ArrayList<>();
+    try {
+      SecureRandom.getInstanceStrong().nextBytes(entropy);
+      passphrase = MnemonicCode.INSTANCE.toMnemonic(entropy);
+    } catch (NoSuchAlgorithmException | MnemonicLengthException e) {
+      e.printStackTrace();
```

Comment:
Should eventually replace these with SLF4J.

Reply:
yeah, i got a `TODO` to fix the slf4j errors. think it is getting imported in bitcoinj or some other dependency and isn't being resolved propertly